index.js

```diff
@@ -1,9 +1,9 @@
-var fs     = require('fs')
-var path   = require('path')
+var fs = require('fs-extra')
```

Comment:
Can we use https://www.npmjs.com/package/ncp instead? `fs-extra` seems excessive since we only need one thing.